builtin/mainmenu/dlg_create_world.lua

```diff
@@ -17,13 +17,115 @@
 
 local worldname = ""
 
+local function flags_to_table(flags)
+	if flags then
+		local ftable = flags:gsub("%s+", ""):split(",", true) -- Remove all spaces and split
+		local ftable2 = {}
+		for f=1, #ftable do
+			if string.sub(ftable[f], 1, 2) == "no" then
+				ftable2[string.sub(ftable[f], 3)] = false
+			else
+				ftable2[ftable[f]] = true
+			end
+		end
+		return ftable2
+	else
+		return {}
+	end
+end
+
+local function table_to_flags(ftable)
+	local str = ""
+	local first = true
+	for flag, is_set in pairs(ftable) do
+		if not first then
+			str = str .. ","
+		end
+		if is_set then
+			str = str .. flag
+		else
+			str = str .. "no" .. flag
+		end
+		first = false
+	end
+	return str
+end
+
+local function chflag(flags, flag)
+	if flags[flag] == true then
+		return "true"
+	else
+		return "false"
+	end
+end
+
+local function chflag2(flags, flag)
+	return flags[flag] == true
+end
+
+local tt_caverns = fgettext("Large cavities will be carved in the ground")
+
+local flag_checkboxes = {
+	v5 = {
+		{ "caverns", fgettext("Caverns"), "caverns", tt_caverns },
+	},
+	v7 = {
+		{ "caverns", fgettext("Caverns"), "caverns", tt_caverns },
+		{ "ridges", fgettext("Rivers"), "ridges",
+		fgettext("Rivers will be aggressively carved through terrain at sea level") },
```

Comment:
No need for 'aggressively', it conveys no essential additional information.

Reply:
Only after reading @Wuzzy2's comment I understood what's meant by "aggressively". It makes sense, but does not provide helpful information what's exactly meant here.

The only suggestion I have here is to remove "aggressively" (as well) because "carved" may already indicate that it'll happen no matter the elevation of the surrounding terrain.